index.js

```diff
@@ -11,6 +11,8 @@ function stringIncludes(haystack, needle) {
 function countLetter(haystack, needle) {
   let count = 0
 
+  haystack = haystack.split('')
```

Comment:
Good idea to create a new variable here rather than overwriting the original haystack.